styles.css

```diff
@@ -99,6 +99,31 @@ main {
     text-transform: uppercase;
 }
 
+.text-capitalize {
+    text-transform: capitalize;
+}
+
+.product-section {
+    gap: 3rem;
+}
+
+.product-header {
+    justify-content: space-between;
+}
+
+.product-section img {
+    width: 30rem;
+}
+
+.product-form {
+    width: 25rem;
+}
+
+.cta-cart-btn,
+.cta-wishlist-btn {
+    width: 100%;
```

Comment:
Instead of using width % property every time you can make col width helper class 

look into these link - https://www.w3schools.com/css/tryit.asp?filename=tryresponsive_cols

Reply:
Nice! Didn't know about this. I will add these helper classes to my CL. Thanks